aptos-move/aptos-vm/src/aptos_vm.rs

```diff
@@ -1900,6 +1982,7 @@ impl AptosSimulationVM {
                         resolver,
                         log_context,
                         &storage_gas_params.change_set_configs,
+                        false,
```

Comment:
Why this is false?

Reply:
As this part of code is executed when a transaction fails, I disabled aggregators here.

Reply:
I see, thanks!